tests/node/rpc/messages/UnsubscribeVehicleDataResponseTests.js

```diff
@@ -65,12 +77,14 @@ describe('UnsubscribeVehicleDataResponseTests', function () {
         const testHandsOffSteering = rpcMessage.getHandsOffSteering();
         const testWindowStatus = rpcMessage.getWindowStatus();
         const testGearStatus = rpcMessage.getGearStatus();
+        const testClimateData = rpcMessage.getClimateData();
 
         // Valid Tests
         Validator.validateVehicleDataResult(this.stabilityControlsStatus, testStabilityControlsStatus);
         Validator.validateVehicleDataResult(this.vehicleDataResult, testHandsOffSteering);
```

Comment:
**Optional:** On line 84 you can change `vehicleDataResult` to `handsOffSteering`.

Reply:
Based on previous experience, it is better to avoid changes not related to PR. Will add if PMs request this in the review.